### ISSUE TITLE
A Question about Gaussion function in SoftMatch

### ISSUE THREAD

**Issue by wzc32** [microsoft/Semi-supervised-learning#97]
Hi, 

I saw a wired parameter in Gaussion function in SoftMax which is not exist in the paper, could anybody tell me what it is？

`mask = torch.exp(-((torch.clamp(max_probs - mu, max=0.0) ** 2) / (2 * var / (self.n_sigma ** 2))))`

what does the parameter `self.n_sigma` here mean?

In the paper or the traditional Gaussion function, the code should be like:

`mask = torch.exp(-((torch.clamp(max_probs - mu, max=0.0) ** 2) / (2 * var)))`

I'd be appreciate it if somebody tell me the meaning.
